model/godleyTable.h

```diff
@@ -154,8 +154,8 @@ namespace minsky
         throw std::out_of_range("Godley table index error");
       return data[row][col];
     }
-    bool cellInTable(int row, int col) const
-    {return row>=0 && size_t(row)<rows() && col>=0 && size_t(col)<cols();}
+    bool cellInTable(unsigned row, unsigned col) const
```

Comment:
I had chosen int deliberately here. If unsigned, the row>=0, col>=0 make no sense.
May be OK - -ve numbers are mapped to >2e9, so row<rows() and col<cols() still works...

Reply:
Ah okay, I understand now. I changed this because I was getting a test build failure with AEGIS=1, but not when I use only TRAVIS=1. So, I can change it back to int? I also now see that you had fixed the problem in the mean time and I had missed it on my last commit after I had pulled the latest remote. I mean you changed 

 {return row>=0 && row<rows() && col>=0 && col<cols();}

to

 {return row>=0 && size_t(row)<rows() && col>=0 && size_t(col)<cols();}

So, I'm changing it back to int with the above fix.